### PR TITLE
Update .htaccess

### PR DESCRIPTION
**Pull request by physta**

### CODE REVIEW

File: frontier/.htaccess [hunk 1]
```diff
@@ -8,57 +8,67 @@ AddType application/rdf+xml .owl
...
-RewriteRule ^def$ doc/index-en.html [R=303,L]
+RewriteEngine on
+
+# Rewrite rule for latest version.
 RewriteCond %{HTTP_ACCEPT} !application/rdf\+xml.*(text/html|application/xhtml\+xml)
 RewriteCond %{HTTP_ACCEPT} text/html [OR]
 RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml [OR]
 RewriteCond %{HTTP_USER_AGENT} ^Mozilla/.*
-RewriteRule ^/(.+) https://physta.github.io/frontier/OnToology/Frontier.ttl/documentation/doc/index-en.html#$1 [R=303,NE,L]
+RewriteRule ^$ https://physta.github.io/frontier/OnToology/Frontier.ttl/documentation/doc/index-en.html [R=303,L]
```
Comment (dgarijo) [inline frontier/.htaccess:19]: This one is redundant with the one in line 46. You can group them by doing:
RewriteRule ^(.*)$ https://physta.github.io/frontier/OnToology/Frontier.ttl/documentation/doc/index-en.html [R=303,L]

**Comment by dgarijo** [comment]
@physta have a look at the comment, please.

**Comment by physta** [comment]
Hello, 

I think that now should be ok. I've used your example template and changed the paths. In your template I see in line 19 
RewriteRule ^$ https://dgarijo.github.io/example/release/1.0.1/index-en.html [R=303,L]
and in line 46
RewriteRule ^(.+)$ https://dgarijo.github.io/example/release/$1/index-en.html [R=303,L]
Is this redundant? Which one I should remove?

**Comment by dgarijo** [comment]
Because I use versions in the URI (see the $1 and the 1.0.1), you are not. Therefore, you can replace both of them by using what I suggested.

**Review by physta — COMMENTED** [review]
Ok thanks, understood. I think that now should be fine

**Comment by dgarijo** [comment]
@physta 
The same applies for the json, ttl and other serializations

**Pull request merged**